chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+which uses a terminology that is more intuitive for users of applied statistics,
+we have added our own twist based on the jargon of development research.
+
+% Definitions
+Development data is typically acquired in the form of \textbf{tables},
+two-dimensional datasets formed by rows and columns.\sidenote{\textbf{Table:}
+	a two-dimensional dataset formed by rows and columns. 
+	Also called ``rectangular data''.
+	One example of non-rectangular data is a text.}
+Each element or cell in this table is a \textbf{data point}, that is,
+a value representing a measure of an attribute for one individual unit.\sidenote{\textbf{Data point:}
+a value representing a measure of an attribute for one individual unit.}
+For example, in a household survey,
+two of the attributes usually measured are gender and age.
+The measures of gender and age for each household member are data points.
+A \textbf{variable}\sidenote{\textbf{Variable:} the collection of all data points that measure the same attribute.}
+ is the collection of all data points that measure a given attribute.
+In the example of a household survey,
+each variable will correspond to one survey question.
+Finally, each variable will contain data points for different individuals in a given unit of observation.
+In our example, each household member is a subject, and the unit of observation is a person.
+The collection of all data points for each individual subject is an 
+\textbf{observation}.\sidenote{\textbf{Observation:} 
+	the collection of all data points representing attributes of the same individual or unit.}
+
+% Tidy data
+The ideal format to interact with any table or dataset is when
+each column represents one variable,
+each row represents one observation, 
+and all variables in a table have the same unit of observation.
+Datasets in this format are said to be \textbf{tidy},\sidenote{\textbf{Tidy data:}
+	Datasets where each column represents one variable,
+	each row represents one observation, 
+	and all variables in a table have the same unit of observation.}
+and all others are \textit{untidy}.
+This may seem trivial, but the raw data acquired is frequently untidy.
+Tidy data sets are easier to handle,
+as observations can be aggregated and linked consistently,
+and easier to clean, as each attribute only needs to be checked once.
+
+% An example of untidy data
+A very simple way for our survey example to became untidy is to include
+household-level variables in the same data set,
+such as whether the household has a refrigerator.
+In this case, some attributes would be measured at
+household level, and others at household member level,
+and it would not be possible for each column to represent a variable
+\textit{and} for each row to represent an observation \textit{at the same time}.
+Either the household-level variable would be repeated for all its members,
+or member-level variables would be stored in 
+\textbf{wide format}.\sidenote{\textbf{Wide data:}
```

Comment:
I think it would be helpful if this comparison comes sooner. As if we are introducing working with data wide vs. working with data tidy are to competing or alternative philosophies. I think we should introduce them as both are commonly used, but that we think tidy is better, and here are tools and concepts for how to successfully adopt the philosophy of tidy.

Reply:
done in the new version